docs/general/ad-filtering/create-own-filters.md

```diff
@@ -530,6 +530,12 @@ AdGuard will try to close the browser tab with any address that matches a blocki
 
 #### **`$third-party`** {#third-party-modifier}
 
+:::note
+
+The `$3p` alias is also used.
```

Comment:
Let's have a common approach to all alias sections.

```
:::note

You may use a shorter name (alias) instead of using the full modifier name: `$3p`.

:::
```

And please put this in the end of the modifier description section.